app/fireweather_bot.py

```diff
@@ -0,0 +1,270 @@
+""" This is a bot to pull hourly weather actuals from the BC FireWeather Phase 1 API
+for each weather station and store the results (from a CSV file) in our database.
+"""
+import os
+import json
+import sys
+import logging
+import logging.config
+import tempfile
+from datetime import datetime, timedelta
+import re
+from typing import List, Dict
+from abc import abstractmethod, ABC
+from requests import Session
+from requests.compat import urljoin
+from requests_ntlm import HttpNtlmAuth
+from sqlalchemy.exc import IntegrityError
+import pandas as pd
+from app import config
+import app.db.database
+from app.db.models import NoonForecasts
+from app.wildfire_one import _get_now, _get_stations_local
```

Comment:
Convention is for underscore on functions to indicate it's to be used internally only - but whatever - point is - we should roll this out into a utils file at some point and rename that _get_now to something that indicates that it's utc at some point. (Not part of this PR)